src/FlexTable.jl

```diff
@@ -243,14 +243,37 @@ end
 # Private fields are never exposed since they can conflict with column names
 Base.propertynames(t::FlexTable, private::Bool=false) = columnnames(t)
 
+
+Base.vcat(t::FlexTable) = t
+
+function Base.vcat(t::FlexTable{<:NamedTuple{names}}, t2::FlexTable{<:NamedTuple{names}}) where {names}
```

Comment:
This is covered below. Plus, this signature isn't right... the only type parameter of a `FlexTable` is dimensionality (`ndims`).

Reply:
Ooops, yes ... copy/paste ... sorry.